installation/db_migration/_migrate_2.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+import argparse
+import asyncio
+import logging
+
+import sqlalchemy as sa
+import sqlalchemy.ext.asyncio as sqlasync
+import sqlalchemy.orm.decl_api
+
+import dso.model
+
+
+Base: sqlalchemy.orm.decl_api.DeclarativeMeta = sqlalchemy.orm.declarative_base()
+
+logger = logging.getLogger(__name__)
+
+
+class ArtefactMetaData(Base):
```

Comment:
should the class name `ArtefactMetaData` be renamed to `ArtefactMetadata` to align with pythons naming conventions?

Reply:
Generally, yes, however I just copied the table definition from [here](https://github.com/open-component-model/delivery-service/blob/f951c9248ea06fc5a95530f10e47f0ac10babe03/deliverydb/model.py#L11) to remove the dependency. I'm not even sure why it was named like that, whether it was a typo or by purpose to differentiate from `dso.model.ArtefactMetadata` 🤔